backend/src/core/models/steam.app.js

```diff
@@ -98,4 +66,70 @@ export class SteamApp {
 
     return steamAppsSource.filter((app) => !targetAppIds.includes(app.appid));
   }
+
+  get appType() {
```

Comment:
no benefit provided by this method if the properties of the class are public, I would remove it